webpack.config.js

```diff
@@ -70,6 +70,9 @@ const config = {
       { from: 'res/_headers' },
       { from: 'res/_redirects' },
       { from: 'docs-user', to: 'docs' },
+      { from: 'res/.htaccess' },
+      { from: 'res/zee-worker.js' },
+      { from: 'res/analytics.js' },
```

Comment:
I realize we need a bit more changes here, for the service worker to work properly.

The service worker is configured below, with the `OfflinePlugin` object. You can see the line below:
```javascript
 externals: ['/zee-worker.js', '/analytics.js'],
```
Now this line is useless because `OfflinePlugin` will find them automatically thanks to your change.

But there's an additional issue: the file `.htaccess` that's picked up automatically by `OfflinePlugin`  should be excluded, so you should add it to the array specified by the property `excludes`.

If you want to know more about the `OfflinePlugin`, here is its homepage: https://github.com/NekR/offline-plugin